docs/root/start/sandboxes/golang_network.rst

```diff
@@ -0,0 +1,72 @@
+.. _install_sandboxes_golang_network:
+
+Golang Network Filter
+=====================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`netcat <start_sandboxes_setup_netcat>`
+        Used to send ``TCP`` data.
+
+In this example, we show how the `Golang <https://go.dev/>`_ network filter can be used with the Envoy
+proxy.
+
+The example demonstrates a Go plugin that can process tcp data stream directly.
+
+It also shows how Go plugins can be independently configured at runtime.
```

Comment:
does it do this ?

Reply:
afaict this sandbox does use a runtime config, so perhaps this comment is valid but then it needs to be highlighted further down how the config is being used

Reply:
I add some description on how the golang plugin retrieve the proxy destination from the configure file.